runners/spark/src/main/java/org/apache/beam/runners/spark/metrics/AggregatorMetricSource.java

```diff
@@ -35,7 +35,7 @@
 
   public AggregatorMetricSource(final String appName, final NamedAggregators aggregators) {
     sourceName = appName;
-    metricRegistry.register("Beam", AggregatorMetric.of(aggregators));
+    metricRegistry.register("Beam.Aggregators", AggregatorMetric.of(aggregators));
```

Comment:
@amitsela 

This change will affect existing aggregators' metrics names in metrics sinks. 
For instance if you use the `GraphiteSink` and you had an aggregator that was reported as `MyApp.Beam.my.aggregator`
after this change it will be reported as `MyApp.Beam.Aggregators.my.aggregator`
Do we need to somehow handle this change?